.pre-commit-config.yaml

```diff
@@ -157,6 +157,8 @@ repos:
     rev: v24.2.0
     hooks:
       - id: ansible-lint
+        additional_dependencies:
```

Comment:
Please see commit [df1e703](https://github.com/cisagov/ansible-role-systemd-resolved/pull/1/commits/df1e703710795d945a87623ae2a9dd53e6909278).

I think the PR you mentioned should perhaps be in [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) since it is there that we first pin `ansible` and `ansible-core`.  What do you think?

Reply:
Well I figured [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic) is appropriate for pinning these for `ansible-lint` so that any project using Ansible will by default respect the pins used by our Ansible roles.

Reply:
Please see cisagov/skeleton-generic#180.

Reply:
As a quick closer question do we want to add `ansible` in the same way but commented out with an explanation about needing to enable it if you run into issues with built-in modules not being found (like here)?

Reply:
Personally I prefer to add it as needed, just to avoid another dependency that needs to be kept in sync across repos.

Reply:
How about just a comment that mentions the possible need if it happens in another downstream repository? That way there's no dependency to maintain but the knowledge of fixing that ansible-lint failure isn't lost.

Reply:
Please see commit cisagov/skeleton-generic@8e57244cbe10b3a3f28276c32395a0228f1d773f.

Reply:
Appreciated!